### PR TITLE
Open registration

### PR DESCRIPTION
**Pull request by zanedb**

### CODE REVIEW

File: src/pages/register.js [hunk 1]
```diff
@@ -22,17 +26,210 @@ const Headline = styled(Heading.h2).attrs({
...
+    <FormField
+      type="text"
+      name="dietary_restrictions"
+      label="Dietary Restrictions"
+      error={errors.dietary_restrictions}
+      value={values.dietary_restrictions}
+      onChange={handleChange}
+    />
+    <FormField
+      type="checkbox" /* not working :( */
```
Comment (lachlanjc) [inline src/pages/register.js]: Hahahaha yeah we don’t support _that_ kind of witchcraft

**Comment by lachlanjc** [comment]
You’re a gorgeous human being, thank you @zanedb

**Comment by zanedb** [comment]
Not so soon, @lachlanjc. It needs a lot of work still :)

File: src/pages/register.js (hunk 1 above)
Comment (zanedb) [reply src/pages/register.js]: Yeahhhhhhh... I'm just using `type="select"` for now.

**Comment by lachlanjc** [comment]
As soon as I picked one option:
fb8b2e85-88d6-49d8-95a2-dc1fcde3c33f

Let’s change all the required() to required(‘required’), & see how to avoid this issue

**Comment by zanedb** [comment]
Fixed the required text. Not sure how to disable the validation after an onChange event.

**Pull request merged**